lib/mix/tasks/config_seed.ex

```diff
@@ -0,0 +1,72 @@
+defmodule Mix.Tasks.ConfigSeed do
```

Comment:
so we won't want a mix task as mix isn't available in releases, we only need a function and you can eval it similar to how we run migrations
```
"./bin/auth", "eval", "Auth.ReleaseTasks.migrate()"`
```

Reply:
Oh ok then I guess I'll turn it into a script exs?

Reply:
ok I did in priv/repo/config_seeds.exs